application/CMakeLists.txt

```diff
@@ -30,6 +30,10 @@ endif()
 add_executable(f3d ${F3D_SOURCE_FILES})
 target_link_libraries(f3d PUBLIC libf3d)
 
+if(UNIX)
```

Comment:
is it required on macos?

Reply:
I do not know if it is needed for us but it may be needed in some cases: https://stackoverflow.com/questions/17841140/os-x-clang-pthread